### DISCUSSION TITLE
Error - `Suspense boundary received an update before it finished hydrating`

### DISCUSSION THREAD

**Discussion by hakimLyon** [remix-run/remix#5312] [Q&A] [answered]
Hi 👋, defer with prisma return undefined ? Pls,  how to solve that ?

![Capture d’écran du 2023-01-30 16-05-12](https://user-images.githubusercontent.com/64654633/215529780-25a247cc-69e1-430d-a582-f66643cd6c37.png)
![Capture d’écran du 2023-01-30 16-08-00](https://user-images.githubusercontent.com/64654633/215530132-bcacd18a-df03-416c-995a-c3754bc70a82.png)

**Comment by machour** [comment, marked as answer]
This have been reported in https://github.com/remix-run/remix/issues/5153
